pkg/util/mount/mount_windows_test.go

```diff
@@ -601,3 +601,89 @@ func TestGetFileType(t *testing.T) {
 		}
 	}
 }
+
+func TestIsLikelyNotMountPoint(t *testing.T) {
+	mounter := Mounter{"fake/path"}
+	base, err := ioutil.TempDir("", "TestIsLikelyNotMountPoint")
+	if err != nil {
+		t.Fatalf(err.Error())
+	}
+
+	tests := []struct {
+		fileName        string
+		fileValid       bool
+		targetLinkName  string
+		targetLinkValid bool
+		expectedResult  bool
+		expectError     bool
+	}{
+		{
+			"Dir",
+			true,
+			"",
+			true,
+			true,
+			false,
+		},
+		{
+			"InvalidDir",
+			false,
+			"",
+			true,
+			true,
+			true,
+		},
+		{
+			"ValidSymLink",
+			true,
+			"targetSymLink",
+			true,
+			false,
+			false,
+		},
+		{
+			"InvalidSymLink",
+			true,
+			"targetSymLink2",
+			false,
+			true,
+			false,
+		},
+	}
+
+	for _, test := range tests {
+		filePath := filepath.Join(base, test.fileName)
+		if test.fileValid {
```

Comment:
You could potentially pass in a setup method per test case, instead of combining all the scenarios here, to make it easier to comprehend each test case separately.

Reply:
@msau42 thanks for the suggestion, I have made a new commit with setUp func, the test code is more clear now